src/PageContent/PendingAccounts/VerifyAccount.js

```diff
@@ -0,0 +1,102 @@
+                    />
+                </div>
+                <div className="address-wrapper">
+                    <h2 className="subheader">{this.props.agencyName}</h2>
+                    <div>
+                        <div className="address">
+                            {this.props.agencyAddressData.street1}
+                            {this.props.agencyAddressData.street2},{' '}
+                            {this.props.agencyAddressData.city}{' '}
+                            {this.props.agencyAddressData.state}{' '}
+                            {this.props.agencyAddressData.zipcode}
+                        </div>
+                        {this.props.agencyAddressData.officeNo && (
+                            <div className="officeNo">
+                                Office: {this.props.agencyAddressData.officeNo}
+                            </div>
+                        )}
+                        <div className="deliveryNote">
+                            {this.props.deliveryNote}
+                        </div>
+                    </div>
+                </div>
+                <div className="agency-wrapper">
+                    <h2 className="subheader">Agency Contact</h2>
+                    <div className="contact">
+                        <div className="contact-value">
+                            {' '}
+                            {this.props.primaryContact}
+                        </div>
+                        <div className="contact-value">
+                            {' '}
+                            {this.props.primaryContactData.position}
+                        </div>
+                        <div className="contact-value gray-contact">
+                            {' '}
+                            {this.props.primaryContactData.email}
+                        </div>
+                        <div className="contact-value gray-contact">
+                            {' '}
+                            {this.props.primaryContactData.phone}
+                        </div>
+                    </div>
+                </div>
+                {this.props.emergencyPickup && (
+                    <div className="emergency-pickup-wrapper">
+                        <img
+                            className="verify-truck-icon"
+                            src={truck}
+                            alt="truck"
+                        />
+                        <div className="verify-emergency-pickup-content">
+                            <h2 className="subheader emergency-pickup-subheader">
+                                Emergency Pickup Activated
+                            </h2>
+                            <div className="emergency-pickup-note">
+                                This agency can pick up in times of emergency.{' '}
+                            </div>
+                        </div>
+                    </div>
+                )}
+                <div className="verification-buttons">
```

Comment:
For both Accept and Reject, I think we should add 2 more screens to the flow:
1. Account Details (what we currently have) [Accept/Reject]

-> click Accept:
2. Are you sure? Once an account is accepted, the user will be able to log in. [Confirm/Back] -> click Confirm
3. Account Accepted. You can now view this account under Directory. [OK]

-> click Reject:
2. Are you sure? The account will immediately be deleted from the system. [Confirm/Back] -> click Confirm
3. Account Rejected. It is now removed from the system. [OK]

Please figure out the exact wording for each step with Madison. For Accept-2, we could also mention something about how once the account is accepted, it can't be removed from the system, it can only be deactivated. Maybe - talk to Madison!